templates/common/infra/bicep/core/host/function-node.bicep

```diff
@@ -0,0 +1,35 @@
+param environmentName string
+param location string = resourceGroup().location
+param serviceName string
+param linuxFxVersion string = 'NODE|16'
+param appSettings object = {}
+param keyVaultName string = ''
+param useKeyVault bool = !(empty(keyVaultName))
+param managedIdentity bool = useKeyVault
+param applicationInsightsName string
+param appServicePlanId string
+param storageAccountName string
+param functionsWorkerRuntime string = 'node'
```

Comment:
Sort of surprised to see this as a parameter, given this module is focused just on node cases (I get that `linuxFxVersion` is exposed because you need it to set the version)

Reply:
Removed